app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class StaticPagesController < ApplicationController
+	def index
```

Comment:
try to watch your spacing on your tabs.  The preference in ruby is to use exactly two spaces to denote a tab.